plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -399,6 +399,16 @@ function ListMenuItem:update()
             else
                 title = bookinfo.title and bookinfo.title or filename_without_suffix
                 authors = bookinfo.authors
+                -- If multiple authors (crengine separates them with \n), we
+                -- can display them on multiple lines, but limit to 2, and
+                -- append "et al." to the 2nd if there are more
+                if authors and authors:find("\n") then
+                    authors = util.splitToArray(authors, "\n")
+                    if #authors > 2 then
+                        authors = { authors[1], authors[2].."  et al." }
```

Comment:
Although et al. will work in many indo-European languages and might've been borrowed in more, it should still be a proper localizable string. ;-)

```lua
T("%1 et al.", authors[2])